README.md

```diff
@@ -1,2 +1,62 @@
-# platform-custom-scalars
-Library of customized GraphQL scalars for Ontotext Platform
+# Ontotext Platform Custom GraphQL Scalars
+A library providing customized scalars used in Ontotext Platform, written in JavaScript
```

Comment:
Also, can we or should we guarantee that when the platform gets changed: add or remove scalars or changes of this sort, this library will be tested and updated respectively, because it could be easily forgotten IMO? I think, a test setup of this kind could be complicated, but still, is it worth to perform such a test? @jazzyray what do you think?

Reply:
@nikolaykolev is writing acceptance tests and I think it will be enough to capture mismatches between the `SOaaS` scalars and the one defined here.
However, if this is not the case, we definitely need to think of something to ensure that there wont be any mismatches.